tezosqa/Dockerfile

```diff
@@ -1,37 +1,28 @@
-FROM python:3.8-slim-bullseye
+FROM docker:20.10.24-dind 
 
-ARG SMARTPY_VERSION
-ARG SMARTPY_PATH
+WORKDIR /tezos
 
-ENV SMARTPY_INSTALL_URL https://smartpy.io/releases/${SMARTPY_VERSION}/cli
-
-RUN smartpy_version_year=$(echo $SMARTPY_VERSION | cut -c 1-4) \
-    && echo "Install system dependencies for python and pip" \
-    && pip install -U pip \
-    && echo "Done base install!" \
+RUN echo "Install python3 and pip3" \
+    && apk  --no-cache --update add python3 py3-pip\
+    && echo "Done!" \
     && echo "Install basics Python tools" \
-    && pip install pipenv \
+    && pip3 install -U pipenv \
     && echo "Done!" \
-    && echo "Install crypto libs" \
-    && apt-get update -q -y && apt-get install -q -y libsodium-dev libsecp256k1-dev libgmp-dev gcc git g++ make \
+    && echo "Install crypto libs and various tools" \
+    && apk add -q libsodium-dev pkgconfig libsecp256k1-dev gmp-dev python3-dev libffi-dev gcc git g++ make wget bash musl-dev linux-headers\
     && echo "Done!" \
     && echo "Install SmartPyDev" \
-    && apt-get install -q -y curl \
-    && curl -sL https://deb.nodesource.com/setup_12.x | bash - \
-    && apt-get install -y nodejs \
-    && curl -s "${SMARTPY_INSTALL_URL}/install.sh" | bash /dev/stdin --prefix ${SMARTPY_PATH} --from ${SMARTPY_INSTALL_URL} --yes \
-    && echo "export PATH=${SMARTPY_PATH}:\$PATH" >> /etc/profile \
+    && wget https://smartpy.io/smartpy \
+    && chmod +x smartpy && mv smartpy /usr/local/bin \
+    && wget smartpy.io/templates/welcome.py\
```

Comment:
Not necessary to download this file or execute the test with it into this Dockerfile.